src/patternfly/components/Page/page.scss

```diff
@@ -383,6 +384,14 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
     --#{$page}__main-container--MaxHeight: var(--#{$page}__main-container--xs--MaxHeight);
     --#{$page}__main-container--BorderRadius: var(--#{$page}__main-container--xs--BorderRadius);
   }
+
+  @media (prefers-contrast: more) {
+    --#{$page}__main-container--BorderWidth: var(--pf-t--global--border--width--strong);
+    --#{$page}__main-container--BorderColor: var(--pf-t--global--border--color--high-contrast);
+    --#{$page}__main-container--Padding--offset: #{pf-size-prem(4px)};
```

Comment:
@lboehling @mcoker We have a comment[ in the code](https://github.com/kmcfaul/patternfly/blob/79019932142721006596fc4e9430642be5be1158/src/patternfly/components/Page/page.scss#L79) saying that we should create page border outline width and color tokens. Is this a good time to do that? Then it could adjust in the high-contrast tokens. Trying to see if we could get rid of this media query! Since the offset is 4px either way, maybe that would do it?

Reply:
I like that! @lboehling do you have any issues with adding that to figma?

Reply:
That would also let us use that token value to offset any component padding/insets that might need to take into consideration that border width to align properly with other page chrome elements. I haven't looked too much into it, but I'm wondering if that might help resolve this issue with tabs not aligning properly with the page chrome gutter https://github.com/patternfly/patternfly/issues/7376